packages/cvc5/cvc5.1.1.3-unreleased/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml bindings for the cvc5 SMT solver"
+description: "OCaml bindings for the cvc5 SMT solver"
+maintainer: ["João Pereira <joaomhmpereira@tecnico.ulisboa.pt>"]
+authors: ["João Pereira <joaomhmpereira@tecnico.ulisboa.pt>"]
+license: "GPL-3.0-only"
+homepage: "https://github.com/formalsec/ocaml-cvc5"
+bug-reports: "https://github.com/formalsec/ocaml-cvc5/issues"
+depends: [
+  "dune" {>= "3.10"}
+  "ocaml" {>= "4.12"}
+  "conf-gcc" {build}
+  "conf-g++" {build}
+  "conf-gmp" {build}
+  "conf-cmake" {build}
+  "conf-python-3" {build}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/formalsec/ocaml-cvc5.git"
+url {
+  src:
+    "https://github.com/formalsec/ocaml-cvc5/archive/refs/tags/v1.1.3-unreleased.tar.gz"
+  checksum: [
+    "md5=2c34475ff7dfd5e92fca5ce1ba1c0d6d"
+    "sha512=cc1cae0cc1cddb2c1bb9fa7978bc43cc3a862c6fc4a8d7f79ae66c08e0af7fe82cac3c00d351e01806b064fce317705870f0a91bd0dafbb0ec06023786186bcb"
```

Comment:
Do a release with all the submodules included. Then, change the link and hashes here